default_config.yaml

```diff
@@ -181,25 +184,111 @@ coaddition:
       measure_psf: true
       threshold: 3.0
       method: sextractor
-      background_method: zero
-    # The following are used to override the regular astrometric calibration parameters
-    wcs:
+    bg:  # override the regular background estimation parameters
+      format: map
+      method: sep
+    wcs:  # override the regular astrometric calibration parameters
       cross_match_catalog: gaia_dr3
       solution_method: scamp
       max_catalog_mag: [22.0]
       mag_range_catalog: 6.0
       min_catalog_stars: 50
-    # The following are used to override the regular photometric calibration parameters
-    zp:
+    zp:  # override the regular photometric calibration parameters
       cross_match_catalog: gaia_dr3
       max_catalog_mag: [22.0]
       mag_range_catalog: 6.0
       min_catalog_stars: 50
 
+# use these parameters to make references by coadding images
+# the reference pipeline will load the regular configuration first,
+# then the coaddition config, and only in the end, override with this:
+referencing:
+  maker:  # these handles the top level configuration (e.g., how to choose images that go into the reference maker)
+    name: best_references  # the name of the reference set that you want to make references for
+    allow_append: true  # can we add different image load criteria to an existing reference set with this name?
+    start_time: null  # only grab images after this time
+    end_time: null  # only grab images before this time
+    instrument: null  # only grab images from this instrument
+    filter: null  # only grab images with this filter
+    project: null  # only grab images with this project
+    min_airmass: null  # only grab images with airmass above this
+    max_airmass: null  # only grab images with airmass below this
+    min_background: null  # only grab images with background rms above this
+    max_background: null  # only grab images with background rms below this
+    min_seeing: null  # only grab images with seeing above this
+    max_seeing: null  # only grab images with seeing below this
+    min_lim_mag: null  # only grab images with limiting magnitude above this
+    max_lim_mag: null  # only grab images with limiting magnitude below this
+    min_exp_time: null  # only grab images with exposure time above this
+    max_exp_time: null  # only grab images with exposure time below this
+    min_number: 7  # only create a reference if this many images can be found
+    max_number: 30  # do not add more than this number of images to the reference
+    seeing_quality_factor: 3.0  # linear coefficient for adding lim_mag and seeing to get the "image quality"
+    save_new_refs: true  # should the new references be saved to disk and committed to the database?
+  pipeline:  # The following are used to override the regular "extraction" parameters
+    extraction:
+      sources:  # override the regular source and psf extraction parameters
+        measure_psf: true
+#        threshold: 3.0
```

Comment:
I've left them in here as a way to demonstrate to potential users what options they have. I think it is better than leaving them with the same values because "same values" tend to change over time and then you wonder why your coadd has a lower threshold, etc. 
I can go either way with removing them or not.